Google.PowerShell/BigQuery/GcbqDataset.cs

```diff
@@ -303,7 +303,7 @@ private class ParameterSetNames
         /// </para>
         /// </summary>
         [Parameter(Mandatory = false, ValueFromPipeline = true, ParameterSetName = ParameterSetNames.ByObject)]
-        public Dataset Dataset { get; set; }
+        public Dataset InputObject { get; set; }
```

Comment:
I'm not sure why this needed to be renamed. "InputObject" is an acceptable generic name, but "Dataset" is a better more specific name. Perhaps an alias is useful?